@@
+ERROR:  cannot specify storage parameters for a partitioned table
+HINT:  Specify storage parameters for its leaf partitions instead.
@@
+ERROR:  relation "bfv_tab2_facttable1" does not exist
+LINE 1: insert into bfv_tab2_facttable1 select col1, col1, col1 from...
+                    ^
@@
+ERROR:  relation "bfv_tab2_facttable1" does not exist
@@
+ERROR:  relation "bfv_tab2_facttable1" does not exist
@@
- count 
--------
-     4
-(1 row)
-
+ERROR:  relation "bfv_tab2_facttable1" does not exist
+LINE 2: FROM bfv_tab2_facttable1 ft, bfv_tab2_dimdate dt, bfv_tab2_d...
+             ^
@@
-QUERY PLAN
-___________
-
+ERROR:  relation "bfv_tab2_facttable1" does not exist
+LINE 2: FROM bfv_tab2_facttable1 ft, bfv_tab2_dimdate dt, bfv_tab2_d...
+             ^
@@
-QUERY PLAN
-___________
-
+ERROR:  relation "bfv_tab2_facttable1" does not exist
+LINE 2: FROM bfv_tab2_facttable1 ft, bfv_tab2_dimdate dt, bfv_tab2_d...
+             ^
@@
+ERROR:  language "plpython3u" does not exist
@@
+ERROR:  table "bfv_tab2_facttable1" does not exist
@@
- count_index_scans 
--------------------
-
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tab23383 whe...
+               ^
+DETAIL:  There is no function of that name.
@@
- 1 | 1
-(1 row)
-
-select count_index_scans('explain select * from Tab23383 where ''1''=b;');
- count_index_scans 
--------------------
@@
+select count_index_scans('explain select * from Tab23383 where ''1''=b;');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tab23383 whe...
+               ^
+DETAIL:  There is no function of that name.
@@
- 1 | 1
-(1 row)
-
-select count_index_scans('explain select * from Tab23383 where ''2''> b order by a limit 10;');
- count_index_scans 
--------------------
@@
+select count_index_scans('explain select * from Tab23383 where ''2''> b order by a limit 10;');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tab23383 whe...
+               ^
+DETAIL:  There is no function of that name.
@@
-  1 | 1
- 10 | 10
- 11 | 11
- 12 | 12
- 13 | 13
- 14 | 14
- 15 | 15
- 16 | 16
- 17 | 17
- 18 | 18
-(10 rows)
-
-select count_index_scans('explain select * from Tab23383 where b between ''1'' and ''2'' order by a limit 10;');
- count_index_scans 
--------------------
@@
+select count_index_scans('explain select * from Tab23383 where b between ''1'' and ''2'' order by a limit 10;');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tab23383 whe...
+               ^
+DETAIL:  There is no function of that name.
@@
-  1 | 1
-  2 | 2
- 10 | 10
- 11 | 11
- 12 | 12
- 13 | 13
- 14 | 14
- 15 | 15
- 16 | 16
- 17 | 17
-(10 rows)
-
--- predicates on both index and non-index key
-select count_index_scans('explain select * from Tab23383 where b=''1'' and a=''1'';');
- count_index_scans 
--------------------
@@
+-- predicates on both index and non-index key
+select count_index_scans('explain select * from Tab23383 where b=''1'' and a=''1'';');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tab23383 whe...
+               ^
+DETAIL:  There is no function of that name.
@@
- 1 | 1
-(1 row)
-
---negative tests: no index scan plan possible, fall back to planner
-select count_index_scans('explain select * from Tab23383 where b::int=''1'';');
- count_index_scans 
--------------------
@@
+--negative tests: no index scan plan possible, fall back to planner
+select count_index_scans('explain select * from Tab23383 where b::int=''1'';');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tab23383 whe...
+               ^
+DETAIL:  There is no function of that name.
@@
- count_index_scans 
--------------------
-
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tbl23383_par...
+               ^
+DETAIL:  There is no function of that name.
@@
- 1 | 1 | 1 | 1
-(1 row)
-
-select count_index_scans('explain select * from Tbl23383_partitioned where ''1''=b');
- count_index_scans 
--------------------
@@
+select count_index_scans('explain select * from Tbl23383_partitioned where ''1''=b');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tbl23383_par...
+               ^
+DETAIL:  There is no function of that name.
@@
- 1 | 1 | 1 | 1
-(1 row)
-
-select count_index_scans('explain select * from Tbl23383_partitioned where ''2''> b order by a limit 10;');
- count_index_scans 
--------------------
@@
+select count_index_scans('explain select * from Tbl23383_partitioned where ''2''> b order by a limit 10;');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tbl23383_par...
+               ^
+DETAIL:  There is no function of that name.
@@
-  1 | 1  | 1  | 1
- 10 | 10 | 10 | 10
- 11 | 11 | 11 | 11
- 12 | 12 | 12 | 12
- 13 | 13 | 13 | 13
- 14 | 14 | 14 | 14
- 15 | 15 | 15 | 15
- 16 | 16 | 16 | 16
- 17 | 17 | 17 | 17
- 18 | 18 | 18 | 18
-(10 rows)
-
-select count_index_scans('explain select * from Tbl23383_partitioned where b between ''1'' and ''2'' order by a limit 10;');
- count_index_scans 
--------------------
@@
+select count_index_scans('explain select * from Tbl23383_partitioned where b between ''1'' and ''2'' order by a limit 10;');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tbl23383_par...
+               ^
+DETAIL:  There is no function of that name.
@@
-  1 | 1  | 1  | 1
-  2 | 2  | 2  | 2
- 10 | 10 | 10 | 10
- 11 | 11 | 11 | 11
- 12 | 12 | 12 | 12
- 13 | 13 | 13 | 13
- 14 | 14 | 14 | 14
- 15 | 15 | 15 | 15
- 16 | 16 | 16 | 16
- 17 | 17 | 17 | 17
-(10 rows)
-
--- predicates on both index and non-index key
-select count_index_scans('explain select * from Tbl23383_partitioned where b=''1'' and a=''1'';');
- count_index_scans 
--------------------
@@
+-- predicates on both index and non-index key
+select count_index_scans('explain select * from Tbl23383_partitioned where b=''1'' and a=''1'';');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tbl23383_par...
+               ^
+DETAIL:  There is no function of that name.
@@
- 1 | 1 | 1 | 1
-(1 row)
-
---negative tests: no index scan plan possible, fall back to planner
-select count_index_scans('explain select * from Tbl23383_partitioned where b::int=''1'';');
- count_index_scans 
--------------------
@@
+--negative tests: no index scan plan possible, fall back to planner
+select count_index_scans('explain select * from Tbl23383_partitioned where b::int=''1'';');
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tbl23383_par...
+               ^
+DETAIL:  There is no function of that name.
@@
- count_index_scans 
--------------------
+ERROR:  function count_index_scans(unknown) does not exist
+LINE 1: select count_index_scans('explain select * from Tbl23383_par...
+               ^
+DETAIL:  There is no function of that name.
+select * from Tbl23383_partitioned where c='1';
+ a | b | c | d 
+---+---+---+---
@@
-select * from Tbl23383_partitioned where c='1';
- a | b | c | d 
----+---+---+---
- 1 | 1 | 1 | 1
-(1 row)
-
@@
+ERROR:  function count_index_scans(text) does not exist
@@
+ERROR:  unrecognized parameter "appendonly"
@@
+ERROR:  syntax error at or near "row"
+LINE 1: ...shape_ao (c circle) with (appendonly=true, orientation=row);
+                                                                  ^
@@
+ERROR:  unrecognized parameter "appendonly"
@@
+ERROR:  relation "shape_heap" does not exist
+LINE 1: insert into shape_heap values ('<(0,0), 5>');
+                    ^
@@
+ERROR:  relation "shape_ao" does not exist
+LINE 1: insert into shape_ao   values ('<(0,0), 5>');
+                    ^
@@
+ERROR:  relation "shape_aocs" does not exist
+LINE 1: insert into shape_aocs values ('<(0,0), 5>');
+                    ^
@@
+ERROR:  relation "shape_heap" does not exist
@@
+ERROR:  relation "shape_ao" does not exist
@@
+ERROR:  relation "shape_aocs" does not exist
@@
- ?column? | ?column? | ?column? 
-----------+----------+----------
- f        | f        | t
-(1 row)
-
+ERROR:  relation "shape_heap" does not exist
+LINE 4: from shape_heap;
+             ^
@@
-QUERY PLAN
-___________
-
+ERROR:  relation "shape_heap" does not exist
+LINE 1: explain select * from shape_heap where c && '<(5,5), 1>'::ci...
+                              ^
@@
-QUERY PLAN
-___________
-
+ERROR:  relation "shape_ao" does not exist
+LINE 1: explain select * from shape_ao   where c && '<(5,5), 1>'::ci...
+                              ^
@@
-QUERY PLAN
-___________
-
+ERROR:  relation "shape_aocs" does not exist
+LINE 1: explain select * from shape_aocs where c && '<(5,5), 1>'::ci...
+                              ^
@@
- c 
----
-(0 rows)
-
+ERROR:  relation "shape_heap" does not exist
+LINE 1: select * from shape_heap where c && '<(5,5), 1>'::circle;
+                      ^
@@
- c 
----
-(0 rows)
-
+ERROR:  relation "shape_ao" does not exist
+LINE 1: select * from shape_ao   where c && '<(5,5), 1>'::circle;
+                      ^
@@
- c 
----
-(0 rows)
-
+ERROR:  relation "shape_aocs" does not exist
+LINE 1: select * from shape_aocs where c && '<(5,5), 1>'::circle;
+                      ^
@@
- c 
----
-(0 rows)
-
+ERROR:  relation "shape_heap" does not exist
+LINE 1: select * from shape_heap where c && '<(5,5), 2>'::circle;
+                      ^
@@
- c 
----
-(0 rows)
-
+ERROR:  relation "shape_ao" does not exist
+LINE 1: select * from shape_ao   where c && '<(5,5), 2>'::circle;
+                      ^
@@
- c 
----
-(0 rows)
-
+ERROR:  relation "shape_aocs" does not exist
+LINE 1: select * from shape_aocs where c && '<(5,5), 2>'::circle;
+                      ^
@@
-     c     
------------
- <(0,0),5>
-(1 row)
-
+ERROR:  relation "shape_heap" does not exist
+LINE 1: select * from shape_heap where c && '<(5,5), 3>'::circle;
+                      ^
@@
-     c     
------------
- <(0,0),5>
-(1 row)
-
+ERROR:  relation "shape_ao" does not exist
+LINE 1: select * from shape_ao   where c && '<(5,5), 3>'::circle;
+                      ^
@@
-     c     
------------
- <(0,0),5>
-(1 row)
-
+ERROR:  relation "shape_aocs" does not exist
+LINE 1: select * from shape_aocs where c && '<(5,5), 3>'::circle;
+                      ^
